README.md

```diff
@@ -108,160 +111,178 @@ See [CoaL Setup](./docs/developer-guide/coal-setup.md) for a thorough walkthroug
 
     1. Launch VMware at least once after installing VMware.
 
-    2. Download and run the following setup script from this repository to
-       configure VMware's networking.
+    2. Run SDC set up script for VMware:
 
          - Mac:
 
             ```bash
-            sudo ./tools/coal-mac-vmware-setup
+            curl -s https://raw.githubusercontent.com/joyent/sdc/master/tools/coal-mac-vmware-setup | sudo bash
```

Comment:
@misterbisson identified this. The existing command has you downloading the whole repo earlier. During one coal set up @tysonhom also found the existing sequence a little confusing.

I appreciate that curl and sudo is considered an anti-pattern by some security types.